### DISCUSSION TITLE
启动器更新失败

### DISCUSSION THREAD

**Discussion by 1234444321qw** [Meloong-Git/PCL#975] [提问] [answered]
![屏幕截图(1)](https://user-images.githubusercontent.com/123076224/213606577-fbfddd02-c38e-44e3-ae9d-827904270648.png)
安装更新失败
Caused By: 自动更新所下载的文件已损坏！
   在 PCL.ModMain._Closure$__112-0._Lambda$__1()
   在 PCL.ModMain._Closure$__112-0._Lambda$__R2(LoaderTask`2 a0)
   在 PCL.ModLoader.LoaderTask`2._Closure$__13-0._Lambda$__1()

**Comment by Louis-Harsune** [comment]
需要完整日志文件，谢谢！

**Comment by quannen** [comment]
把杀软关了（没有的话就把WD的实时防护关了

**Comment by LTCatt** [comment, marked as answer]
建议直接下载最新版，不要用自动更新了，这个版本实在是太老了……
